lib/modules/manager/cargo/artifacts.ts

```diff
@@ -17,6 +17,7 @@ import { extractLockFileContentVersions } from './locked-version';
 async function cargoUpdate(
   manifestPath: string,
   isLockFileMaintenance: boolean,
+  userConfiguredEnv: Record<string, string> = {},
```

Comment:
```suggestion
  userConfiguredEnv: Record<string, string>,
```
I don't think we should pass a default value